### DISCUSSION TITLE
help run blockscout

### DISCUSSION THREAD

**Discussion by Ranofty** [blockscout/blockscout#6234] [Support]
Hello please i need help to run blockscout am already done with all the processes but can't open it on browser
below are my logs 

 sudo systemctl start explorer.service

sudo systemctl status explorer.service

● explorer.service - chainscan Server
     Loaded: loaded (/etc/systemd/system/explorer.service; enabled; vendor pres>
     Active: activating (auto-restart) (Result: exit-code) since Sun 2022-10-09>
    Process: 89264 ExecStart=/usr/local/bin/mix phx.server (code=exited, status>
   Main PID: 89264 (code=exited, status=1/FAILURE)

Oct 09 09:09:14 ip-172-31-68-184 systemd[1]: explorer.service: Failed with resu>
lines 1-7/7 (END)

sudo journalctl -u explorer.service -f

45: Config.__import__!/1
Oct 09 09:11:45 ip-172-31-68-184 mix[92502]:     (stdlib 3.17.1) erl_eval.erl:685: :erl_eval.do_apply/6
Oct 09 09:11:45 ip-172-31-68-184 mix[92502]:     (stdlib 3.17.1) erl_eval.erl:123: :erl_eval.exprs/5
Oct 09 09:11:45 ip-172-31-68-184 mix[92502]:     (elixir 1.13.4) lib/code.ex:404: Code.validated_eval_string/3
Oct 09 09:11:45 ip-172-31-68-184 mix[92502]:     (stdlib 3.17.1) erl_eval.erl:685: :erl_eval.do_apply/6
Oct 09 09:11:45 ip-172-31-68-184 systemd[1]: explorer.service: Main process exited, code=exited, status=1/FAILURE
Oct 09 09:11:45 ip-172-31-68-184 systemd[1]: explorer.service: Failed with result 'exit-code'.
Oct 09 09:11:46 ip-172-31-68-184 systemd[1]: explorer.service: Scheduled restart job, restart counter is at 1416.
Oct 09 09:11:46 ip-172-31-68-184 systemd[1]: Stopped chainscan Server.
Oct 09 09:11:46 ip-172-31-68-184 systemd[1]: Started chainscan Server.
Oct 09 09:11:47 ip-172-31-68-184 mix[92540]: ** (File.Error) could not read file "/usr/local/blockscout/apps/block_scout_web/config/prod# run in production mode.exs": no such file or directory
Oct 09 09:11:47 ip-172-31-68-184 mix[92540]:     (elixir 1.13.4) lib/file.ex:355: File.read!/1
Oct 09 09:11:47 ip-172-31-68-184 mix[92540]:     (elixir 1.13.4) lib/config.ex:245: Config.__import__!/1
Oct 09 09:11:47 ip-172-31-68-184 mix[92540]:     (stdlib 3.17.1) erl_eval.erl:685: :erl_eval.do_apply/6
Oct 09 09:11:47 ip-172-31-68-184 mix[92540]:     (stdlib 3.17.1) erl_eval.erl:123: :erl_eval.exprs/5
Oct 09 09:11:47 ip-172-31-68-184 mix[92540]:     (elixir 1.13.4) lib/code.ex:404: Code.validated_eval_string/3
Oct 09 09:11:47 ip-172-31-68-184 mix[92540]:     (stdlib 3.17.1) erl_eval.erl:685: :erl_eval.do_apply/6
Oct 09 09:11:47 ip-172-31-68-184 systemd[1]: explorer.service: Main process exited, code=exited, status=1/FAILURE
Oct 09 09:11:47 ip-172-31-68-184 systemd[1]: explorer.service: Failed with result 'exit-code'.
Oct 09 09:11:48 ip-172-31-68-184 systemd[1]: explorer.service: Scheduled restart job, restart counter is at 1417.
Oct 09 09:11:48 ip-172-31-68-184 systemd[1]: Stopped chainscan Server.
Oct 09 09:11:48 ip-172-31-68-184 systemd[1]: Started chainscan Server.
Oct 09 09:11:49 ip-172-31-68-184 mix[92578]: ** (File.Error) could not read file "/usr/local/blockscout/apps/block_scout_web/config/prod# run in production mode.exs": no such file or directory
Oct 09 09:11:49 ip-172-31-68-184 mix[92578]:     (elixir 1.13.4) lib/file.ex:355: File.read!/1
Oct 09 09:11:49 ip-172-31-68-184 mix[92578]:     (elixir 1.13.4) lib/config.ex:245: Config.__import__!/1
Oct 09 09:11:49 ip-172-31-68-184 mix[92578]:     (stdlib 3.17.1) erl_eval.erl:685: :erl_eval.do_apply/6
Oct 09 09:11:49 ip-172-31-68-184 mix[92578]:     (stdlib 3.17.1) erl_eval.erl:123: :erl_eval.exprs/5
Oct 09 09:11:49 ip-172-31-68-184 mix[92578]:     (elixir 1.13.4) lib/code.ex:404: Code.validated_eval_string/3
Oct 09 09:11:49 ip-172-31-68-184 mix[92578]:     (stdlib 3.17.1) erl_eval.erl:685: :erl_eval.do_apply/6
Oct 09 09:11:49 ip-172-31-68-184 systemd[1]: explorer.service: Main process exited, code=exited, status=1/FAILURE
Oct 09 09:11:49 ip-172-31-68-184 systemd[1]: explorer.service: Failed with result 'exit-code'.
Oct 09 09:11:50 ip-172-31-68-184 systemd[1]: explorer.service: Scheduled restart job, restart counter is at 1418.
Oct 09 09:11:50 ip-172-31-68-184 systemd[1]: Stopped chainscan Server.
Oct 09 09:11:50 ip-172-31-68-184 systemd[1]: Started chainscan Server.
Oct 09 09:11:50 ip-172-31-68-184 mix[92616]: ** (File.Error) could not read file "/usr/local/blockscout/apps/block_scout_web/config/prod# run in production mode.exs": no such file or directory
Oct 09 09:11:50 ip-172-31-68-184 mix[92616]:     (elixir 1.13.4) lib/file.ex:355: File.read!/1
Oct 09 09:11:50 ip-172-31-68-184 mix[92616]:     (elixir 1.13.4) lib/config.ex:245: Config.__import__!/1
Oct 09 09:11:50 ip-172-31-68-184 mix[92616]:     (stdlib 3.17.1) erl_eval.erl:685: :erl_eval.do_apply/6
Oct 09 09:11:50 ip-172-31-68-184 mix[92616]:     (stdlib 3.17.1) erl_eval.erl:123: :erl_eval.exprs/5
Oct 09 09:11:50 ip-172-31-68-184 mix[92616]:     (elixir 1.13.4) lib/code.ex:404: Code.validated_eval_string/3
Oct 09 09:11:50 ip-172-31-68-184 mix[92616]:     (stdlib 3.17.1) erl_eval.erl:685: :erl_eval.do_apply/6
Oct 09 09:11:50 ip-172-31-68-184 systemd[1]: explorer.service: Main process exited, code=exited, status=1/FAILURE
Oct 09 09:11:50 ip-172-31-68-184 systemd[1]: explorer.service: Failed with result 'exit-code'.
Oct 09 09:11:52 ip-172-31-68-184 systemd[1]: explorer.service: Scheduled restart job, restart counter is at 1419.
Oct 09 09:11:52 ip-172-31-68-184 systemd[1]: Stopped chainscan Server.
Oct 09 09:11:52 ip-172-31-68-184 systemd[1]: Started chainscan Server.
Oct 09 09:11:52 ip-172-31-68-184 mix[92654]: ** (File.Error) could not read file "/usr/local/blockscout/apps/block_scout_web/config/prod# run in production mode.exs": no such file or directory
Oct 09 09:11:52 ip-172-31-68-184 mix[92654]:     (elixir 1.13.4) lib/file.ex:355: File.read!/1
Oct 09 09:11:52 ip-172-31-68-184 mix[92654]:     (elixir 1.13.4) lib/config.ex:245: Config.__import__!/1
Oct 09 09:11:52 ip-172-31-68-184 mix[92654]:     (stdlib 3.17.1) erl_eval

sudo netstat -tulpn

Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 127.0.0.53:53           0.0.0.0:*               LISTEN      2869/systemd-resolv
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      794/sshd: /usr/sbin
tcp6       0      0 :::22                   :::*                    LISTEN      794/sshd: /usr/sbin
udp        0      0 0.0.0.0:37999           0.0.0.0:*                           22146/avahi-daemon:
udp        0      0 0.0.0.0:5353            0.0.0.0:*                           22146/avahi-daemon:
udp        0      0 127.0.0.53:53           0.0.0.0:*                           2869/systemd-resolv
udp        0      0 172.35.62.187:69        0.0.0.0:*                           2863/systemd-networ
udp6       0      0 :::37399                :::*                                22146/avahi-daemon:
udp6       0      0 :::5353                 :::*                                22146/avahi-daemon:

i need your help on this please

**Comment by mrandall101** [comment]
How did you fix this issue?
